app/controllers/admin/batches_controller.rb

```diff
@@ -25,38 +25,31 @@ def show
   end
 
   def create
-    @council = current_user.council
-
-    @rebate_forms = policy_scope(RebateForm)
-                    .joins(:property)
-                    .where(status: RebateForm::SIGNED_STATUS,
-                           properties: { council: @council },
-                           batch: nil)
-                    .order(:created_at)
-                    .limit(100)
-
-    if @rebate_forms.size.positive?
-      @batch = Batch.new(
-        council: current_user.council,
-        amount: @rebate_forms.sum(:rebate),
-        batch_date: @rebate_forms.last.created_at,
-        claim_count: @rebate_forms.size
-      )
-      authorize @batch
-
-      Batch.transaction do
-        @batch.save
-        @rebate_forms.each do |form|
-          form.update(batch: @batch)
-        end
+    rebate_forms = params[:ids].map { |id| policy_scope(RebateForm).find(id) }
+
+    batch = Batch.new(
+      council: rebate_forms.first.council,
```

Comment:
Are we sure these all have the same council?

Reply:
@Br3nda I pondered that for a while - they SHOULD do, in terms of the business logic as I understand it.

But maybe there is a better way to go about this/enforce it?

Reply:
If you are the admin user and you make a batch, will it make a batch for all the councils?

Reply:
i think the policy_scope is gonna save us from letting the use create batches with rebate forms from councils they shouldn't see.

I'll give it a good test

Reply:
All of the rebate_forms will need to match the council_id of the batch.

Reply:
@Br3nda Cool, let me know what you think. Happy to make any changes.